cli/src/commands/rebase.rs

```diff
@@ -290,13 +290,27 @@ fn rebase_descendants(
     rebase_options: RebaseOptions,
 ) -> Result<(), CommandError> {
     workspace_command.check_rewritable(old_commits)?;
+    let (skipped_commits, old_commits) = old_commits
+        .iter()
+        .partition::<Vec<_>, _>(|commit| commit.parents().eq(new_parents));
+    for commit in skipped_commits.iter() {
+        writeln!(
+            ui.stderr(),
+            "Skipping rebase of commit {}",
+            short_commit_hash(commit.id()),
+        )?;
+        continue;
+    }
```

Comment:
Might want to extract a function for this? I think we can use it in the `rebase_revision()` case too.